photonVision/cameras/Microsoft_LifeCam_HD-3000/pipelines/Cargo Red.json

```diff
@@ -0,0 +1,86 @@
+[ "ColoredShapePipelineSettings", {
+  "pipelineIndex" : 1,
+  "pipelineType" : "ColoredShape",
+  "inputImageFlipMode" : "NONE",
+  "inputImageRotationMode" : "DEG_0",
+  "pipelineNickname" : "Cargo Red",
+  "cameraExposure" : 49.1,
+  "cameraBrightness" : 50,
+  "cameraGain" : -1,
+  "cameraVideoModeIndex" : 0,
+  "streamingFrameDivisor" : "NONE",
+  "ledMode" : true,
+  "inputShouldShow" : true,
+  "outputShouldShow" : true,
+  "hsvHue" : {
+    "first" : 0,
+    "second" : 20
+  },
+  "hsvSaturation" : {
+    "first" : 80,
+    "second" : 200
+  },
+  "hsvValue" : {
+    "first" : 114,
+    "second" : 255
+  },
+  "outputShouldDraw" : true,
+  "outputShowMultipleTargets" : true,
+  "contourArea" : {
+    "first" : 1.5,
+    "second" : 100.0
+  },
+  "contourRatio" : {
+    "first" : 0.0,
+    "second" : 20.0
+  },
+  "contourFullness" : {
+    "first" : 0.0,
+    "second" : 100.0
+  },
+  "contourSpecklePercentage" : 51,
+  "contourSortMode" : "Largest",
+  "contourTargetOffsetPointEdge" : "Center",
+  "contourTargetOrientation" : "Landscape",
+  "offsetRobotOffsetMode" : "None",
+  "offsetSinglePoint" : {
+    "x" : 0.0,
+    "y" : 0.0
+  },
+  "offsetDualPointA" : {
+    "x" : 0.0,
+    "y" : 0.0
+  },
+  "offsetDualPointAArea" : 0.0,
+  "offsetDualPointB" : {
+    "x" : 0.0,
+    "y" : 0.0
+  },
+  "offsetDualPointBArea" : 0.0,
+  "contourGroupingMode" : "Single",
+  "contourIntersection" : "Up",
+  "solvePNPEnabled" : false,
+  "targetModel" : "k2020HighGoalOuter",
+  "cornerDetectionStrategy" : "APPROX_POLY_DP_AND_EXTREME_CORNERS",
+  "cornerDetectionUseConvexHulls" : true,
+  "cornerDetectionExactSideCount" : false,
+  "cornerDetectionSideCount" : 4,
+  "cornerDetectionAccuracyPercentage" : 10.0,
+  "contourShape" : "Circle",
+  "contourPerimeter" : {
+    "first" : 0.0,
+    "second" : 4000.0
+  },
+  "accuracyPercentage" : 10.0,
+  "circleDetectThreshold" : 15,
+  "contourRadius" : {
+    "first" : 10,
+    "second" : 10
+  },
+  "minDist" : 20,
+  "maxCannyThresh" : 1,
+  "circleAccuracy" : 10,
```

Comment:
> Circle Accuracy - This determines how perfect the circle contour must be in order to be considered a circle. Low values (<40) are required to detect things that aren’t perfect circles.

We need to have this be a high number. Greater than 40. 

When we filter for the balls, our HSV filtering should be wide enough that we're getting nearly "perfect" circle shape in the filtered camera view.

Once we have nearly "perfect" balls filtered, the Circle Accuracy being a high number will help _only_ detect circle shaped filtered blobs.